client/extensions/woocommerce/state/sites/settings/general/reducer.js

```diff
@@ -53,6 +54,10 @@ export default createReducer( null, {
 		return LOADING;
 	},
 
+	[ WOOCOMMERCE_SETTINGS_GENERAL_REQUEST_FAILURE ]: () => {
```

Comment:
Is this needed? The initial state should be null, so we shouldn't actually need to react here. On other potential pages where we might be fetching settings, if settings were previously loaded, but a new request fails, the state would be removed. We might want to keep it around until this request passes.